### PR TITLE
feat: enable proxy caching in off2 nginx proxy

### PR DESCRIPTION
**Pull request by stephanegigandet**
This PR enables proxy caching in off2 nginx proxy for some requests like /api/ and small static assets like CSS, JS and some images like icons.

### CODE REVIEW

File: confs/proxy-off/nginx/openfoodfacts.org [hunk 1]
```diff
@@ -43,9 +43,60 @@ server {
     access_log  /var/log/nginx/openfoodfacts.org.log  main buffer=256K flush=1s;
     error_log   /var/log/nginx/openfoodfacts.org.errors.log;
 
+    # Cache small static assets that are frequently requested
+
+
+    location ~ ^/(css/|js/|fonts/|images/(attributes|favicon|icons|illustrations|lang|logos|misc|panels|svg)/|.well-known/|api/v./(preferences|attribute_groups)|data/i18n) {
+    	proxy_cache mycache;
```
Comment (alexgarel) [inline confs/proxy-off/nginx/openfoodfacts.org:50]: Can we name it more sensibly ? (off-cache)

**Review by alexgarel — APPROVED** [review]
LGTM appart from the cache name !

**Comment by alexgarel** [comment]
@stephanegigandet when I do live updates on the servers, most of the time I directly commit them, it's better than having complications with branch on the server.

File: confs/proxy-off/nginx/nginx.conf [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+include /etc/nginx/modules-enabled/*.conf;
+
+events {
+	worker_connections 768;
+	# multi_accept on;
+}
+
+http {
+	# Cache
+	proxy_cache_path /dev/shm/nginx-cache levels=1:2 keys_zone=mycache:10m max_size=100m inactive=2m use_temp_path=off;
```
[the remark this answers is not in the document]
Comment (stephanegigandet) [reply confs/proxy-off/nginx/nginx.conf:13]: @alexgarel it's automatically created by nginx

**Comment by stephanegigandet** [comment]
> @stephanegigandet when I do live updates on the servers, most of the time I directly commit them, it's better than having complications with branch on the server.

OK. But then it means we don't have PRs and reviews tracked for them...

**Pull request merged**